### PR TITLE
Fixing Climsoft dialog widget bug

### PR DESCRIPTION
**Pull request by maxwellfundi**
@dannyparsons  I have added the button onto the sdgClimsoft that leads to dlgOptions where we have a new Climsoft tab with the fields- Database name, Port, Host and username.

### CODE REVIEW

**Comment by dannyparsons** [comment]
This wasn't the suggestion as I understood it. I thought the button on the climsoft subdialog was to save the current values as the new dialog "defaults" (which can also be set in the options dialog)? The button means you can do this without going to the options dialog.

**Comment by maxwellfundi** [comment]
Okay, this is was my understanding. if that is the case then its just changing its the text there.

**Comment by maxwellfundi** [comment]
@africanmathsinitiative/developers  this is ready for review.

**Comment by maxwellfundi** [comment]
@Patowhiz   the bug described on issue #4915 other improvements will be done later.

File: instat/sdgImportFromClimSoft.vb [hunk 1]
```diff
@@ -121,4 +126,23 @@ Public Class sdgImportFromClimSoft
             cmdEnterPassword.Text = "Enter Password"
         End If
     End Sub
+
+    Private Sub cmdOptions_Click(sender As Object, e As EventArgs) Handles cmdOptions.Click
+        Dim msgSetDefaults = MsgBox("Are you sure you want to update Climsoft defaults?", MessageBoxButtons.YesNo, "Factory Reset")
+
+        If msgSetDefaults = DialogResult.Yes Then
+            If Not ucrInputDatabaseName.IsEmpty() Then
```
Comment (dannyparsons) [inline instat/sdgImportFromClimSoft.vb]: I think actually this should change, even when its empty it should be set as the default. You may want to clear the default and this prevents it.

**Pull request merged**